src/services/NoticeService.js

```diff
@@ -11,15 +11,19 @@ class NoticeServiceError extends ExtendableError {}
  * @returns
  */
 const createNotice = async data => {
-  const { type, body, attributes, priority, user } = data;
+  const { type, body, attributes, priority, user, toUser } = data;
```

Comment:
I noticed the connection notice was assigning the toUser, but I dont think it was being used.